src/setupProxy.js

```diff
@@ -2,8 +2,7 @@ const { createProxyMiddleware } = require('http-proxy-middleware')
 
 module.exports = (app) => {
   app.use(
-    '/api/',
-    createProxyMiddleware({
```

Comment:
Does this change anything or is it just style? (Either is fine, just wondering.)

Reply:
`/api/` definition works fine - but `/jmws` did not work otherwise. Wanted to be consistent and changed the `/api/` route as well. Maybe there is something else to be done - I really just left it as it is as soon as it worked.

Reply:
Was just wondering what the difference in the APIs was. But if it works it works! :)